README.md

```diff
@@ -72,6 +72,21 @@ By default, the application will use `mongoqp-cache/` within the system's
 temporary directory. This path, which must be writable, may be customized via
 the `twig.cache_dir` configuration option.
 
+#### Docker Setup 
+Clone the repo
+
+You just have to add the mongo uri in config.php.dist and run commands : 
+```
+$ docker build -t mongoqp .
+```
+
+then run it using 
+
+```
+$ docker run -p 8080:8080 mongoqp
```

Comment:
It's clear you're setting up port forwarding here. I'm not familiar with Docker's networking configuration, but is there any risk that the application within the container would have trouble connecting to a MongoDB server running outside of it (as is likely to be the case for most all users)?

Since the driver's default connection URI is `"mongodb://127.0.0.1:27017"`, I'm wondering if that will look for a MongoDB server within the Docker container or outside the container on the host machine. If the default URI points to a server within the container, what URI should be used to connect to a server on the host machine? I don't think we need to change the defaults, but this might be something worth documenting if it's likely to be a common question.